x-pack/plugins/siem/public/containers/source/index.tsx

```diff
@@ -115,4 +115,4 @@ export class WithSource extends React.PureComponent<WithSourceProps> {
 }
 
 export const indicesExistOrDataTemporarilyUnavailable = (indicesExist: boolean | undefined) =>
-  isUndefined(indicesExist) ? false : indicesExist;
+  indicesExist || isUndefined(indicesExist);
```

Comment:
Why? This will load the page if there is an `undefined` index, causing errors. Needs to return false if index is `undefined`